### PR TITLE
Web app updates

### PR DESCRIPTION
**Pull request by m-r-q-w-e-r-t-y**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -17,6 +17,9 @@ <h1>Anthony's Portfolio</h1>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+
+      <button onclick="showtime()">Click here</button>
```
Comment (atsao72) [inline portfolio/src/main/webapp/index.html:21]: The function you've defined is actually called showServerTime(), make sure to use that here

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,12 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+//Using the showtime() function
+async function showServerTime(){
+    const responseFromServer = await fetch('/date');
+    const textFromResponse = await responseFromServer.text();
+
+    const showtimecontainer = document.getElementById('showtime-container');
```
Comment (atsao72) [inline portfolio/src/main/webapp/script.js:35]: It's usually idiomatic to use camel case, so something like showTimeContainer.

**Review by atsao72 — CHANGES_REQUESTED** [review]
Great PR. Nice and small! 

There is one small suggestion, and one change that I think is a bug. Please resolve and then I can approve and you can merge.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (m-r-q-w-e-r-t-y) [reply portfolio/src/main/webapp/script.js:35]: Updated showtimecontainer to showTimeContainer

**Pull request merged**